xbmc/addons/binary-addons/AddonInstanceHandler.cpp

```diff
@@ -124,7 +124,7 @@ ADDON_STATUS IAddonInstanceHandler::CreateInstance()
   return status;
 }
 
-void IAddonInstanceHandler::DestroyInstance()
+void IAddonInstanceHandler::DestroyInstance() const
```

Comment:
I don't think this makes logically sense, this obviously modifies the object.

Reply:
I agree, while const-correct it's not context-correct

Reply:
Looking at it this way, I have to agree. :-) Thanks for pointing this out.